### PR TITLE
Admob App Open Ad Integration

### PR DESCRIPTION
**Pull request by mhanzla80**

### CODE REVIEW

File: lib/src/easy_ads.dart [hunk 1]
```diff
@@ -185,6 +193,18 @@ class EasyAds {
 
       await ad.load();
     }
+
+    if (appOpenAdUnitId != null &&
+        _appOpenAds.doesNotContain(AdNetwork.admob, AdUnitType.appOpen)) {
+      final appOpenAdManager = EasyAdmobAppOpenAd(
+          appOpenAdUnitId, _adRequest, AppOpenAd.orientationPortrait);
```
Comment (nooralibutt) [inline lib/src/easy_ads.dart:200]: Orientation user sy leni chahie

File: example/ios/Flutter/AppFrameworkInfo.plist [hunk 2]
```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>9.0</string>
+  <string>11.0</string>
```
Comment (nooralibutt) [inline example/ios/Flutter/AppFrameworkInfo.plist:24]: Yeh bhi mention kro change doc main

File: example/android/app/build.gradle [hunk 3]
```diff
@@ -56,4 +56,11 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    constraints {
+        implementation('com.facebook.android:audience-network-sdk') {
```
Comment (nooralibutt) [inline example/android/app/build.gradle:60]: Yeh bhi mention kro change doc main

**Review by nooralibutt — CHANGES_REQUESTED** [review]

File: lib/src/easy_ads.dart (hunk 1 above)
Comment (mhanzla80) [reply lib/src/easy_ads.dart]: added

File: example/ios/Flutter/AppFrameworkInfo.plist (hunk 2 above)
Comment (mhanzla80) [reply example/ios/Flutter/AppFrameworkInfo.plist:24]: mentioned in readme.

File: example/android/app/build.gradle (hunk 3 above)
Comment (mhanzla80) [reply example/android/app/build.gradle]: removed.

File: lib/src/easy_ads.dart [hunk 4]
```diff
@@ -345,13 +368,19 @@ class EasyAds {
...
-        adUnitType == AdUnitType.interstitial ||
-            adUnitType == AdUnitType.rewarded,
-        'Only interstitial and rewarded types should be passed to this method');
-
-    final List<EasyAdBase> ads =
-        adUnitType == AdUnitType.rewarded ? _rewardedAds : _interstitialAds;
+    // assert(
+    //     adUnitType == AdUnitType.interstitial ||
+    //         adUnitType == AdUnitType.rewarded,
+    //     'Only interstitial and rewarded types should be passed to this method');
```
Comment (nooralibutt) [inline lib/src/easy_ads.dart:374]: Remove these

**Review by nooralibutt — CHANGES_REQUESTED** [review]

**Review by nooralibutt — APPROVED** [review]

**Pull request merged**